fix for failing xtb test:
 72/148 dftd4 / param                           FAIL              1.85s   exit status 1

from meson-logs/testlog.txt:
# Testing: param
  Starting rational-damping ... (1/1)
       ... rational-damping [FAILED]
  Message: Condition not fullfilled

see https://github.com/grimme-lab/xtb/pull/1085
diff --git a/subprojects/dftd4.wrap b/subprojects/dftd4.wrap
index 46dc07ad9..e3e93f459 100644
--- a/subprojects/dftd4.wrap
+++ b/subprojects/dftd4.wrap
@@ -1,4 +1,4 @@
 [wrap-git]
 directory = dftd4
 url = https://github.com/dftd4/dftd4
-revision = v3.5.0
+revision = v3.6.0
